squad/frontend/templates/squad/project.html

```diff
@@ -9,21 +10,58 @@
 {% include "squad/project-nav.html" %}
 
 {% if last_build %}
+<div ng-app='SquadBuilds' ng-controller='BuildsController'>
 <h2>Last build - {% if not last_build.status.finished %}<i title="Build/testing in progress" class="fa fa-cog fa-spin"></i>{% endif %}
+<i class="fa fa-exclamation-triangle text-danger" ng-show="regressions[{{last_build.id}}] != null" id="popperRegressions{{ last_build.id }}_last"></i>
+<i class="fa fa-check-circle text-success popper" ng-show="fixes[{{last_build.id}}] != null" id="popperFixes{{ last_build.id }}_last" ></i>
 <a class="h2 text-primary" href="{% project_url last_build %}">{{last_build.version}}</a> {{last_build.datetime}} {{last_build.datetime|naturaltime}}</h2>
+<div class="hidden">
+	<div id="regressions{{ last_build.id }}_last">
+		<div ng-repeat="(regression_key, regression_value) in regressions[{{ last_build.id }}]">
+			<h3><span ng-bind="regression_key"></span></h3>
+			<ul>
+				<li ng-repeat="test in regression_value"><span ng-bind="test"></span></li>
+			</ul>
+		</div>
+	</div>
+	<div id="fixes{{ last_build.id }}_last">
+		<div ng-repeat="(fix_key, fix_value) in fixes[{{ last_build.id }}]">
+			<h3><span ng-bind="fix_key"></span></h3>
+			<ul>
+				<li ng-repeat="test in fix_value"><span ng-bind="test"></span></li>
+			</ul>
+		</div>
+	</div>
+</div>
+
 {% include "squad/_metadata.html" %}
 
 <h2>Latest builds</h2>
 {% include "squad/_builds_table.html" %}
+</div>
 {% endif %}
 
 {% endwith %}
 {% endblock %}
 
 {% block javascript %}
 <script>
+var REGRESSIONS = new Array();
+var FIXES = new Array();
+{% for status in statuses %}
+    {% if status.regressions is not None %}
+        REGRESSIONS[{{ status.build.id }}] = {{ status.regressions|yaml2json }};
+    {% endif %}
+    {% if status.fixes is not None %}
+        FIXES[{{ status.build.id }}] = {{ status.fixes|yaml2json }};
+    {% endif %}
+{% endfor %}
+</script>
+<script type="text/javascript" src='{% static "squad/builds.js" %}'></script>
+<script>
```

Comment:
ditto wrt not being necessary if the list of regressions or fixes is generated server side